util/filesystem.go

```diff
@@ -43,20 +43,38 @@ var dfHeaderPattern = regexp.MustCompile(
 
 var logger = logging.GetLogger("util.filesystem")
 
+var dfOpt = "-Pkl"
+
+func init() {
+	// Some `df` command such as busybox does not have `-P` option.
```

Comment:
We should mention absence of `-l` too.
(Maybe busybox does not implement `-l` at all?)

Reply:
Yes. The `df` command on my busybox environment also does not implement the `-l` option.